app/page-partials/my-dashboard/TeamInfoTabs.tsx

```diff
@@ -240,13 +233,13 @@ function TeamInfoTabs({ alert, currentUser, team, loadTeams }: Props) {
   const [loading, setLoading] = useState(false);
   const [deleteMemberId, setDeleteMemberId] = useState<number>();
   const [modalType, setModalType] = useState('allow');
-  const canDeleteServiceAccount = integrations.length === 0;
   const openModal = () => (window.location.hash = addMemberModalId);
   const inProgressServiceAccount = serviceAccount?.status !== 'applied' && !serviceAccount?.archived;
-  const showDeleteServiceAccountModal = () => (window.location.hash = deleteServiceAccountModalId);
+  const showDeleteServiceAccountModal = () => {
+    window.location.hash = deleteServiceAccountModalId;
+  };
 
   const handleDeleteSeviceAccount = async () => {
```

Comment:
typo (note on the pr, but  service has an 'r')

Reply:
good catch, fixed it